demo/lib/host.bash

```diff
@@ -18,6 +18,22 @@ host-require-govm() {
     command -v "$GOVM" >/dev/null || error "cannot run govm \"$GOVM\". Check PATH or set GOVM=/path/to/govm."
 }
 
+host-get-vm-config() {
+    if [ -z "$1" ]; then
+        error "can't get VM configuration, name not set"
+    fi
+    VM_NAME="$1"
+    HOST_VM_DATA_DIR="$(eval "echo $HOST_VM_DATA_DIR_TEMPLATE")"
+    VM_DATA_CONFIG="$HOST_VM_DATA_DIR/vm-config"
+    if ! [ -f "$VM_DATA_CONFIG" ]; then
+        return 1
+    fi
+    source "$VM_DATA_CONFIG"
+    if [ -z "$VM_DISTRO" ] || [ -z "$VM_CRI" ] || [ -z "$VM_SSH_USER" ]; then
```

Comment:
For consistently/completeness, should we check for VM_NAME, too?

Reply:
Thanks, I agree it's better when complete, as `VM_NAME` is saved to the config file.

Little things that were and still are difficult to decide: should `VM_NAME` be in the config or not? This is why I struggled checking it, too. Anyway, it makes sense to check everything we have there.

Reply:
> should `VM_NAME` be in the config or not?

I was pondering the same thing, but, have no idea 😄  I think this good, now